Sources/WeaverCodeGen/Codable.swift

```diff
@@ -95,6 +100,7 @@ extension ConfigurationAttributeName: Codable {
         case setter = "set"
         case escaping = "e"
         case platforms = "p"
+        case projects = "j"
```

Comment:
Ahh J the most famous letter in the work "Projects" lol

Reply:
haha